config.md

```diff
@@ -28,7 +28,8 @@ Changing it means creating a new derived image, instead of changing the existing
 A layer DiffID is a SHA256 digest over the layer's uncompressed tar archive and serialized in the descriptor digest format, e.g., `sha256:a9561eb1b190625c9adb5a9513e72c4dedafc1cb2d4c5236c9a6957ec7dfd5a9`.
 Layers must be packed and unpacked reproducibly to avoid changing the layer DiffID, for example by using tar-split to save the tar headers.
 
-NOTE: the DiffID is different than the layer digest because it is taken over the uncompressed gzipped layer for `application/vnd.oci.image.layer.v1.tar+gzip` types.
+The difference between DiffIDs and the layer digests in the [manifest's `layers`](manifest.md#image-manifest-property-descriptions) is that the layer digest is taken over the blob regardless of compression, while the DiffID is taken after removing any compression.
```

Comment:
> Assert the positive notion (what a DiffID is taken over)…

This happens in the preceding paragraph (“A layer DiffID is…”).

> …then qualify it.

This paragraph is not attempting to qualify what a DiffID is.  It is just underlining the difference between what a DiffID is (defined in the previous paragraph) and what a layer digest is (defined in `manifest.md`).  A given uncompressed blob may have multiple compressed forms, but “the uncompressed blob associated with this potentially compressed blob” is unambiguous, so it made more sense to me to work it in the layer digest → DiffID direction.

However, if you would prefer different wording for this section, I'm happy to push a commit with whatever you want.  Just tell me what it needs to be in a comment here or file a PR against my branch.